packages/vite/types/stylus.d.ts

```diff
@@ -0,0 +1,1778 @@
+// Project: https://github.com/stylus/stylus
```

Comment:
**issue (blocking):** Why is this file in vite repo needed?
May we just add the `@types/stylus` to `devDependencies`?
(https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/stylus)

Reply:
Because there has a error https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/stylus/index.d.ts#L661 that filename is optional.

Reply:
Then you should do two things:

1. Fix it in the definitely typed repo
2. Use e.g. `// @ts-expect-error: descriptive comment` as a workaround
   This will trigger an error when the types got updated

Reply:
The solution you suggested is better.